source/codegen/metadata_validation.py

```diff
@@ -404,6 +414,8 @@ def _get_function_enums(functions_metadata: dict) -> Set[str]:
         for param in function["parameters"]:
```

Comment:
This function is probably duplicated somewhere right? Is it possible to share the code?

Reply:
Oh yeah, found it in common_helpers. Now calling `common_helpers.get_function_enums(metadata["functions"], metadata["enums"])` to get the list of function enums. It also already accounted for 'force-include' enums so I removed that check in `validate_enum`.